include/swift/AST/DiagnosticsSema.def

```diff
@@ -4788,6 +4788,9 @@ ERROR(nonisolated_local_var,none,
 ERROR(nonisolated_actor_sync_init,none,
       "'nonisolated' on an actor's synchronous initializer is invalid",
       ())
+ERROR(nonisolated_wrapped_property,none,
+      "'nonisolated' is not supported on wrapped properties",
```

Comment:
I have intentionally chose "is not supported" rather than "can not be applied" because we could potentially support it in the future. Does that sound good?

Reply:
I'm not sure how I feel about `wrapped properties` here. WDYT, @hborla?

Reply:
`'nonisolated' is not supported on properties with property wrappers`

Reply:
Thank you! I picked up "wrapped properties" from the documentation, but this is certainly clearer. I have updated the diagnostic now.